gremlin-scala/src/test/scala/gremlin/scala/TraversalSpec.scala

```diff
@@ -871,11 +871,20 @@ class TraversalSpec extends WordSpec with Matchers {
         val michael = g.addV(Person).property(Name -> "michael").head
         val karlotta = g.addV(Person).property(Name -> "karlotta").head
 
-        g.V(michael).addE("loves").to(karlotta).head
-        g.addE("knows").from(michael).to(karlotta).head
+        g.V(michael).addE("loves").to(karlotta).iterate()
```

Comment:
Just a QQ on why you're switching from head -> iterate()?

Reply:
it's doing the same thing (executing the traversal), but we don't care about it's results